StudyBuddy/StudyBuddy.csproj

```diff
@@ -12,7 +12,8 @@
     </ItemGroup>
 
     <ItemGroup>
-      <Folder Include="wwwroot\avatars\" />
+        <Folder Include="NewFolder\" />

```

Comment:
What is this for?

Reply:
tbh I can't even find this on my project